integration/util.py

```diff
@@ -117,8 +117,8 @@ def _cleanup_twistd_process(twistd_process, exited):
     :return: After the process has exited.
     """
     try:
-        print("signaling {} with KILL".format(twistd_process.pid))
-        twistd_process.signalProcess('KILL')
+        print("signaling {} with TERM".format(twistd_process.pid))
+        twistd_process.signalProcess('TERM')
```

Comment:
I'm curious about whether this KILL was the cause of the mysterious unavailable coverage results or if there was some other explanation.

Reply:
It was the cause of the "half of the processes don't get coverage", yes -- and also I'd changed this (and a bunch of other things) and then reverted back but forgot about this piece :/ so tried a bunch of OTHER stuff too. Anyway, this is definitely part of the reason I went for "explicit is better than implicit" and have Tahoe start + stop coverage .. I'm going to add an option though.